# parameters for robots
robot:
  trajectory_length: 12
  maximum_velocity: 10.0

# parameters for g2o optimizer
optimizer:
  maximum_iteration: 200
  verbose: false

# parameters for topic subscription
topic:
  range: /ltp_tag0/nlnf3
  imu: /vn100/imu_ #for ntu sequences
  # imu: /vn200/imu  #for kth and tuhh

publish_flag:
  tf: true
  # publish estimation when the following topics are received.
  # topics available in {range, pose, twist, imu}. false in default
  range: true
  imu: false

# fused pose topic frame
frame:
  target: /uwb_imu
  source: /world
